samples/modules/lvgl/demos/boards/da1469x_dk_pro.overlay

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright (c) 2023 Renesas Electronics Corporation
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+ #include <zephyr/dt-bindings/gpio/gpio.h>
+ #include <zephyr/dt-bindings/display/panel.h>
+
+/ {
+    chosen {
+        zephyr,display = &display;
+    };
+
+    lvgl_pointer {
+        input = <&display_touch>;
+        status = "okay";
+        swap-xy;
+    };
+};
+
+&dma {
+    status = "okay";
+};
+
+&i2c2 {
+    clock-frequency = <400000>;
+
+    display_touch: ft6206@38 {
+        compatible = "focaltech,ft5336";
+        status = "okay";
+        reg = <0x38>;
+        int-gpios = <&gpio0 31 GPIO_ACTIVE_LOW>;
+    };
+};
+
+&display {
+    disp-gpios = <&gpio0 25 GPIO_ACTIVE_HIGH>;
+    status = "okay";
+    width = <480>;
+    height = <272>;
+    pixel-format = <PANEL_PIXEL_FORMAT_RGB_565>;
+
+    /* Display settings for ST7282T2 conttoller */
+    display-timings {
```

Comment:
Might be worth adding a comment here describing the display model (IE part number) you are using, so others know which module has been tested with this sample

Even better, adding documentation to the board `index.rst` highlighting which display modules are validated with the EVK would probably help quite a bit

Reply:
I'd prefer to update board's `index.rst` once the memory controller driver [PR](https://github.com/zephyrproject-rtos/zephyr/pull/68023) is merged (required as most of the display panels resolutions exceed device's system RAM) so I can add all the display part numbers I have tested with.